### PR TITLE
Save send-runtime binary as an artifact

### PR DESCRIPTION
**Pull request by pmikolajczyk41**

### CODE REVIEW

File: .github/workflows/save-send-runtime-binary.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          toolchain: ${{ env.RUST_TOOLCHAIN_VERSION }}
+          override: true
+
+      - name: Build binary
+        run: |
+          pushd local-tests/send-runtime/
+          cargo build --release
+          popd
+
+      - name: Upload binary
```
Comment (h4nsu) [inline .github/workflows/save-send-runtime-binary.yml:48]: Why are we uploading the "bare" binary as an artifact?

File: .github/workflows/save-send-runtime-binary.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        uses: actions/upload-artifact@v2
+        with:
+          name: send-runtime
+          path: local-tests/send-runtime/target/release/send_runtime
+          if-no-files-found: error
+          retention-days: 7
+
+      - name: Build send-runtime-image
```
Comment (h4nsu) [inline .github/workflows/save-send-runtime-binary.yml:56]: I find the name of this image somewhat confusing. How about `runtime-update-hook`? Or `runtime-update-image`?

**Review by h4nsu — APPROVED** [review]

File: .github/workflows/save-send-runtime-binary.yml (hunk 2 above)
Comment (pmikolajczyk41) [reply .github/workflows/save-send-runtime-binary.yml]: I took the former proposition

File: .github/workflows/save-send-runtime-binary.yml (hunk 1 above)
Comment (pmikolajczyk41) [reply .github/workflows/save-send-runtime-binary.yml]: leftover from previous approach; removed

**Pull request merged**